installation/install.sh

```
@@ -983,10 +983,10 @@ cs_create_repo(){
 
 cs_check_python_installed(){
     # ansible1.9.6 is depended on by python2
-    which python2 >/dev/null 2>&1
-    [ $? -ne 0 ] && yum --disablerepo=* --enablerepo=zstack-local install -y python2 >/dev/null 2>&1
-    [ ! -f /usr/bin/python -a -f /usr/bin/python2 ] && ln -s /usr/bin/python2 /usr/bin/python
-    [ ! -f /usr/bin/easy_install -a -f /usr/bin/easy_install-2 ] && ln -s /usr/bin/easy_install-2 /usr/bin/easy_install
+    which python3.11 >/dev/null 2>&1
+    [ $? -ne 0 ] && yum --disablerepo=* --enablerepo=zstack-local install -y python3.11 >/dev/null 2>&1
+    [ ! -f /usr/bin/python -a -f /usr/bin/python3.11 ] && ln -s /usr/bin/python3.11 /usr/bin/python
```

Shellcheck (0.10.0)

[warning] Prefer [ p ] && [ q ] as [ p -a q ] is not well defined.

(SC2166)